src/editors/containers/VideoEditor/components/VideoSettingsModal/components/hooks.js

```diff
@@ -205,15 +205,15 @@ export const arrayWidget = ({ dispatch, key }) => {
   const widget = module.valueHooks({ dispatch, key });
   return {
     ...widget,
-    onChange: handleIndexTransformEvent({
+    onBlur: handleIndexTransformEvent({
```

Comment:
this is an alphabetical change? want to be sure I'm reading it right

Reply:
yes